qemu/tests/balloon_check.py

```diff
@@ -165,7 +169,8 @@ def _mem_state(self):
         while True:
             yield stable
             cur_mem = self.get_memory_status()
-            stable = abs(cur_mem - ori_mem) < 100
+            stable = abs(cur_mem - ori_mem) < self.params.get(
+                     "guest_stable_threshold", 100)
```

Comment:
The function should not get the value by `params`, instead, the value should be provided by the caller.
```python
def _mem_state(self, threshold):
    ...

def wait_for_balloon_complete(self, timeout):
    ...
    is_stable = self._mem_state(threshold)
    ...
```

Reply:
Modified,thanks